kogito-editors-js/packages/boxed-expression-component/src/components/Table/Table.tsx

```diff
@@ -180,15 +186,15 @@ export const Table: React.FunctionComponent<TableProps> = ({
       if (DEFAULT_ON_ROW_ADDING !== rowFactory && !isLockedTable) {
         const pastedRows = pasteOnTable(pasteValue, rows, rowFactory, x, y);
         tableRows.current = pastedRows;
-        onRowsUpdate?.(pastedRows);
+        onRowsUpdate?.(pastedRows, undefined, undefined, columns);
```

Comment:
I feel like this function should have an object parameter. It's unclear to me what those two sequential `undefined`s mean. Can you please change the signature of this method to receive a single object-parameter instead of 4 separate parameters?

Reply:
Yes, sure!